config.mk

```diff
@@ -27,9 +27,11 @@ endef
 # TAG is the git tag or branch to checkout
 # Projects will be started in this order
 define SUBPROJECT_REPOS
-git@github.com:/reactioncommerce/reaction-hydra.git,reaction-hydra,v2.9.0 \
-git@github.com:/reactioncommerce/reaction.git,reaction,v2.9.0 \
-git@github.com:/reactioncommerce/example-storefront.git,example-storefront,v2.9.0
+git@github.com:/reactioncommerce/example-storefront.git,example-storefront,v3.0.0-alpha \
+git@github.com:/reactioncommerce/reaction-admin.git,reaction-admin,v3.0.0-alpha \
+git@github.com:/reactioncommerce/reaction-hydra.git,reaction-hydra,v3.0.0-alpha \
+git@github.com:/reactioncommerce/reaction-indentity.git,reaction-indentity,v3.0.0-alpha \
```

Comment:
small typo here, the cloning will fail as it wont be able to find the `indentity` repo

Reply:
Great catch, thank you